vrecord

```diff
@@ -1142,7 +1134,7 @@ _writeingestlog "computer_name" "$(uname -n)"
 _writeingestlog "user_name" "$(whoami)"
 _writeingestlog "operating_system_version" "$(uname -v)"
 _writeingestlog "datetime_start" "$(_get_iso8601)"
-_writeingestlog "video_bit_depth" "${video_bitdepth}"
+_writeingestlog "video_bit_depth" "${pixel_format}"
```

Comment:
Change also the labelling from `video_bit_depth` to `pixel_format `?

Reply:
Hmm, the issue here is whether the use picks between `8` and `10` or between `yuv422p10` or `uyvy422`.

Reply:
Indeed, yet in my opinion – not blocking at all! – it does not hurt if called what is actually is.